src/array_api_stubs/_draft/linalg.py

```diff
@@ -793,7 +793,7 @@ def vector_norm(
     *,
     axis: Optional[Union[int, Tuple[int, ...]]] = None,
     keepdims: bool = False,
-    ord: Union[int, float, Literal[inf, -inf]] = 2,
+    ord: Union[int, float, Literal[inf, -inf]] = 2,  # type: ignore
```

Comment:
Perhaps this is out of scope, but this `Literal` is redundant (and as this change implies, also incorrect), since `inf` and `-inf` are both assignable to `float`.